cypress/component/helptopics/HelpTopicManager.cy.tsx

```diff
@@ -47,6 +47,11 @@ const initialScalprumConfig = {
     appId: 'TestApp',
     manifestLocation: '/foo/bar.json',
   },
+  virtualAssistant: {
+    name: 'virtualAssistant',
```

Comment:
@justinorringer since we had to touch so many of these specs that setup the scalprum context - it may be nice to move this to a shared setup as well (in case we need to remove it in the future, or add more site wide modules)